app/src/main/java/me/guillaumin/android/osmtracker/db/DataHelper.java

```diff
@@ -65,6 +68,9 @@
 	 */
 	private Context context;
 
+	/** Statistics for all existing tracks */
+	private static TreeMap<Long, TrackStatistics> tracksStatistics = new TreeMap<Long, TrackStatistics> ();
```

Comment:
I want to store it because creating a new instance of TrackStatistics iterates throigh all points in the track, an I don't want that to happen every second when the active track is updated. Thing is, four different classes (including this one) need to share this data, so if it should be a field of a class, I have to ask: which one?